src/sentry_plugins/heroku/plugin.py

```diff
@@ -45,9 +67,9 @@ def handle(self, request: Request) -> Response:
                     "email": email,
                 },
             )
-        self.finish_release(
-            version=request.POST.get("head_long"), url=request.POST.get("url"), owner=user
-        )
+        commit = data.get("slug", {}).get("commit")
+        app_name = data.get("app", {}).get("name")
```

Comment:
I can do that, I've actually been trying to track down what the point of passing `url` to a release is. I don't see it mentioned in our release docs and I don't see it displayed on the release page anywhere. I mostly just did this to maintain consistency but it might not be important.

Reply:
Maybe I can just conditionally pass it to `finish_release` if it exists or not